HW3/task_2.py

```diff
@@ -0,0 +1,6 @@
+my_name= "Andrii"
+age_in_month = 288
+age_in_years = age_in_month / 12
+my_age = (f"My name is {my_name} I'm {age_in_years} years old")
```

Comment:
nit: brackets can be ommitted
```suggestion
my_age = f"My name is {my_name} I'm {age_in_years} years old"
```